src/python/grpcio_test/setup.py

```diff
@@ -43,13 +52,25 @@
         'credentials/server1.pem',]
 }
 
-_INSTALL_REQUIRES = ['oauth2client>=1.4.7', 'grpcio>=0.10.0a0']
+_SETUP_REQUIRES = (
+    'pytest>=2.6',
+    'pytest-cov>=2.0',
+    'pytest-xdist>=1.11',
+)
+
+_INSTALL_REQUIRES = (
+    'oauth2client>=1.4.7', 'grpcio>=0.10.0a0'
+)
 
 setuptools.setup(
     name='grpcio_test',
-    version='0.0.1',
+    version='0.10.0a0',
```

Comment:
Does this matter at all or are we keeping it in sync with grpcio's version number purely out of an aesthetic sense of symmetry?